baselines/showcase-esm/esm/src/v1beta1/echo_client.ts.baseline

```diff
@@ -459,6 +461,9 @@ export class EchoClient {
         protos.google.showcase.v1beta1.IEchoRequest|undefined, {}|undefined
       ]>|void {
     request = request || {};
+    if (!request.requestId) {
+      request.requestId = gax.makeUUID();
```

Comment:
Could we have a generated unit test that would check that this code is actually executed? It would be similar to an existing test that makes a request, but in a stub, it can check that the field was indeed assigned to some UUID.  Similar to https://github.com/googleapis/gapic-generator-typescript/blob/2a61552f77d302f15bccabfaded698f1e2b07e12/templates/esm/typescript_gapic/esm/test/gapic_%24service_%24version.ts.njk#L319.